packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel.tsx

```diff
@@ -273,12 +316,35 @@ export interface NotificationsPanelProps {
 interface PreviousStateProps {
   open: boolean;
 }
+
+/**
+ * The `NotificationsPanel` sets expectations on the behavior for notifications,
+ * allowing the user to view and interact with them all in one place.
+ *
+ * #### Updated localization feature for the "time ago" timestamp
+ *
+ * The new localization reduces a dev's effort to apply language
+ * support for the "time ago" timestamp applied to each notification
+ * by using the new Carbon utility [dateTimeFormat()](https://github.com/carbon-design-system/carbon/blob/main/packages/utilities/src/dateTimeFormat/README.md) function.
+ * It will provide relative timestamps that adhere to Carbon's
+ * [date and time guidance](https://pages.github.ibm.com/carbon/ibm-products/guidelines/content/date-and-time/).
+ *
+ * **Step 1:** ignore the following _deprecated_ props: `daysAgoText`,
+ * `hourAgoText`, `hoursAgoText`, `minuteAgoText`, `minutesAgoText`,
+ * `monthAgoText`, `monthsAgoText`, `nowText`, `secondsAgoText`,
+ * `yearAgoText`, `yearsAgoText`, `yesterdayAtText`.
+ *
+ * **Step 2:** provide the `dateTimeLocale` prop with a valid locale—such as "de" or "fr-CA"—and it will do the work.
+ * You must provide a value, otherwise the deprecated props above will be used instead.
+ */
```

Comment:
I think we’d typically just include this in the actual documentation (StoryDocs or mdx) rather than in the component file.

I would consider reversing the order for adopting the new props though. Rather than “ignore” as a task, I think it makes more sense to 1) adopt, 2) remove.

> To adopt the new localization:
>
> **Step 1:** Provide a valid locale to the `dateTimeLocale` prop, such as `de` or `fr-CA`.
> **Step 2:** Remove the now _deprecated_ props: `daysAgoText`, `hourAgoText`, `hoursAgoText`, `minuteAgoText`, `minutesAgoText`, `monthAgoText`, `monthsAgoText`, `nowText`, `secondsAgoText`, `yearAgoText`, `yearsAgoText`, `yesterdayAtText`.
>
> If you do not provide a locale, the deprecated props will be applied instead.

Reply:
`<StoryDocsPage />` is picking up this comment and dropping into the Overview section on the **Docs** page. I can't find an example of telling `<StoryDocsPage />` to fill in the Overview section directly.

Reply:
I actually imagine we’ll eventually move back to MDX through the docs epic 👀, but for now, I’d just focus on revising the docs you added rather than unpacking the `StoryDocsPage`. I want to say that was a workaround when we ran into issues with MDX when we first moved to Storybook 7 or 8? 🤔

Reply:
Works for me 👍 I'll stop digging into `StoryDocsPage` if we're planning to move back to `mdx`.